testsuite/MDAnalysisTests/topology/test_tprparser.py

```diff
@@ -55,7 +55,7 @@ def test_moltypes(self, top):
     def test_molnums(self, top):
         molnums = top.molnums.values
         assert_equal(molnums, self.ref_molnums)
-        assert molnums.dtype == np.int64
+        assert molnums.dtype == np.intp
```

Comment:
While nobody likes changing unit tests to make them pass, I think the proposed approach (from Richard and maybe Lily?) for portability is that integer indexing will now be platform-sized rather than coerced  to `int64`, so some tests will have platform-specific typing.